docs/docs/dynamo-store/architecture.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_position: 1
+---
+
+# DynamoStore Architecture
+
+When it comes to building an event store on top of a serverless NoSQL database
+it is imperative to apply mechanical sympathy. We've seen implementations of
+DynamoDB event stores using a single-event-per-row strategy. This would seem the
+obvious choice so let's start by explaining why it's not the greatest idea.
+
+## DynamoDB cost model
+
+A great article to understand DynamoDB's pricing model was written by Zach
```

Comment:
Zach Charles has a great [article conveying the structure of DynamoDB's pricing model](https://zaccharles.medium.com/calculating-a-dynamodb-items-size-and-consumed-capacity-d1728942eb7c) - the charges closely mirror the actual work involved, and any good design will balance those forces well.